runelite-client/src/main/java/net/runelite/client/plugins/info/InfoPanel.java

```diff
@@ -105,51 +130,102 @@ void init()
 			}
 		});
 
+
+		versionPanel.add(version);
+		versionPanel.add(revision);
+		versionPanel.add(Box.createGlue());
+		versionPanel.add(loggedLabel);
+		versionPanel.add(emailLabel);
+
 		updateLoggedIn();
 
-		final JRichTextPane issueLink = new JRichTextPane("text/html",
-			"RuneLite is open source!<br>"
-			+ "Found an issue? Want a feature?<br>"
-			+ "<a href=\"https://github.com/runelite/runelite/issues\">"
-			+ "Open an issue on GitHub!"
-			+ "</a>"
-		);
-
-		setBorder(BorderFactory.createEmptyBorder(2, 6, 6, 6));
-
-		layout.setVerticalGroup(layout.createSequentialGroup()
-			.addGroup(layout.createParallelGroup()
-				.addComponent(runeliteVersionHeader)
-				.addComponent(runescapeVersionHeader)
-			).addGroup(layout.createParallelGroup()
-				.addComponent(runeliteVersion)
-				.addComponent(runescapeVersion)
-			).addGap(6)
-			.addComponent(usernameHeader)
-			.addGroup(layout.createParallelGroup()
-				.addComponent(username)
-			)
-			.addGap(12)
-			.addComponent(issueLink)
-		);
-
-		layout.setHorizontalGroup(layout.createParallelGroup()
-			.addGroup(layout.createSequentialGroup()
-				.addComponent(runeliteVersionHeader)
-				.addPreferredGap(LayoutStyle.ComponentPlacement.UNRELATED, GroupLayout.PREFERRED_SIZE, Short.MAX_VALUE)
-				.addComponent(runescapeVersionHeader))
-			.addGroup(layout.createSequentialGroup()
-				.addComponent(runeliteVersion)
-				.addPreferredGap(LayoutStyle.ComponentPlacement.UNRELATED, GroupLayout.PREFERRED_SIZE, Short.MAX_VALUE)
-				.addComponent(runescapeVersion))
-			.addComponent(usernameHeader)
-			.addComponent(username)
-			.addComponent(issueLink)
-		);
+		JPanel actionsContainer = new JPanel();
+		actionsContainer.setBorder(new EmptyBorder(10, 0, 0, 0));
+		actionsContainer.setOpaque(false);
+		actionsContainer.setLayout(new GridLayout(3, 1, 0, 10));
+
+		actionsContainer.add(buildLinkPanel("github_icon.png", "Report an issue or", "make a suggestion", "https://github.com/runelite"));
+		actionsContainer.add(buildLinkPanel("discord_icon.png", "Talk to us on our", "discord server", "https://discordapp.com/invite/R4BQ8tU"));
+		actionsContainer.add(buildLinkPanel("patreon_icon.png", "Become a patron to", "help support RuneLite", "https://www.patreon.com/runelite"));
+
+
+		add(versionPanel, BorderLayout.NORTH);
+		add(actionsContainer, BorderLayout.CENTER);
 
 		eventBus.register(this);
 	}
 
+	/**
+	 * Builds a link panel with a given icon, text and url to redirect to.
+	 */
+	private JPanel buildLinkPanel(String iconName, String topText, String bottomText, String url)
+	{
```

Comment:
There are these extra "new lines" in multiple files, or after a brace. IMO, there shouldn't need to be more than 1 blank line in places, and there shouldn't be a blank line after a brace. That's a reason to have braces on a new line (it already adds whitespace to the code). Those should probably be removed.

As an example for multiple new lines, in this file, line 132, 133 and 150, 151 can just be one line.